modules/command/publish/insert-assertion-command.js

```diff
@@ -15,22 +15,13 @@ class InsertAssertionCommand extends Command {
      * @param command
      */
     async execute(command) {
-        const {
-            documentPath, keywords, assets, handlerId, operationId,
-        } = command.data;
-
-        const { rdf, assertion } = await this.fileService.loadJsonFromFile(documentPath);
+        const { documentPath, handlerId } = command.data;
+        let { nquads, assertion } = await this.fileService.loadJsonFromFile(documentPath);
 
         try {
-            // Store to local graph database
-            await this.dataService.insert(rdf.join('\n'), `did:dkg:${assertion.id}`);
-            this.logger.info(`Assertion ${assertion.id} is successfully inserted`);
-            // await Models.assertions.create({
-            //     owner: '',
-            //     hash: assertion.id,
-            //     signature: '',
-            //     topics: keywords.join(','),
-            // });
+            // TODO Store to local graph database
+            await this.dataService.insert(nquads.join('\n'), `did:dkg:${assertion.id}`);
```

Comment:
Move 'did:dkg' string inside of the constants

Reply:
We are using it only on the graph level, so I guess that it is easier to have it only there

Reply:
But we are using it throughout the code in send-assertion-command, send-telemetry-command, query-service, publish-service, that's why i think that it's not only on the graph level